My-Spec-Tutorial.package/MyFirstWindow.class/instance/setupTitleRadioButtons.st

```diff
@@ -1,11 +1,12 @@
-as yet unclassified
+private
 setupTitleRadioButtons
         radioMr label: 'Mr.'.
         radioMs label: 'Ms.'.
         radioMrs label: 'Mrs.'.
         
-        RadioButtonGroup new
+        titleOptions := RadioButtonGroup new
```

Comment:
The button group is the object who will be able to answer the question "which button is selected?", so we need a reference to ask it later...